DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan_suite/test_basic_functionality.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.utils.test_utils.tgen_utils import tgen_utils_get_dent_devices_with_tgen
+from DentOS_Framework.DentOsTestbed.src.dent_os_testbed import TestBed
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+@pytest.mark.asyncio
+async def test_vlan_basic_functionality(testbed: TestBed):
+    """
+    Test Name: VLAN basic functionality
+    Test Suite: suite_functional_vlan
+    Test Overview: Test VLAN tagged packets can be received/not received on specified port.
+    Test Procedure:
+    1.Specify test data: packet to send, port settings: vla id, tagged/untagged etc.
+        Port 1 vlan 1 pvid untagged
+        Port 2 vlan 1 pvid untagged
+        Port 3 vlan 1 pvid untagged
+        Port 4 vlan 2 untagged
+
+    2.Create bridge entity and set to "up" state.
+        Verify bridge entity created and is set to "up" state.
+
+    3.Create links and set state to `up`.
+        Verify links are created and put to "ùp" state.
+
+    4.Set bridge as master to links.
+        Verify links enslaved to bridge
+
+    5.Set links to vlans as specified in step #1.
+        Verify vlans successfully added to links
+
+    6. Map receiving and non receiving ports. Who sends and who not.
+        Link 1 ->  tx_port
+        Links 2-4 -> rx_ports
+
+    7. Setup packet stream for the broadcast package:

```

Comment:
package or packet?
Why broadcast only? you have test_untagged_broadcast.

Reply:
Why broadcast only?
because based on configuration we sending untagged packet via port_1 and checking that port 2 and 3 has traffic, port 4 do not